cmd/config.go

```diff
@@ -48,6 +48,7 @@ var configFile = os.Getenv("K6_CONFIG") // overridden by `-c` flag!
 
 // configFileFlagSet returns a FlagSet that contains flags needed for specifying a config file.
 func configFileFlagSet() *pflag.FlagSet {
+	//TODO: remove?
```

Comment:
No, that's what I'll investigate today. More details on the whole issue here: https://github.com/loadimpact/k6/issues/883#issuecomment-468646291

Reply:
I think it's mostly because the options are in different FlagSets, apparently that's not a conflict or an error for some reason... Anyway, will clean this up and leave only the global config flag in `root.go`, since that's the one that's shown to users anyway, I'll just point it towards the `configFile` variable. And I'll hopefully make the whole thing more testable as well